modules/nw-ipfailover-virtual-ip-addresses-concept.adoc

```diff
@@ -15,5 +15,5 @@ Keepalived on each node determines whether the needed service is running. If it
 
 [NOTE]
 ====
-Each VIP in the set may end up being served by a different node.
+Each VIP in the set might be served by a different node.
```

Comment:
```suggestion
Each VIP in the set can be served by a different node.
```
Not sure if this is a pre-existing condition that we have to deal with it now ("might be"); or a condition that we are about to potentially create ("can be").